src/pages/Account/AccountInfo/Index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {Stack, Typography, Box} from "@mui/material";
+import Card from "../../../components/Card";
+
+type AccountInfoProps = {
+  address: string;
+};
+
+// TODO: fetch balances data
+export default function AccountInfo({address}: AccountInfoProps) {
+  const cardProps = {marginX: 2};
+
+  return (
+    <Box>
+      <Card {...cardProps}>
+        <Stack direction="column" spacing={2}>
+          <Typography variant="subtitle1">APT Balance: 123456</Typography>
```

Comment:
Do we intend to ship this placeholder text? Keep in mind that this will be live as soon as you merge this PR.

Reply:
This won't be alive with the merge of this PR. Please see the context in Summary: The new account page is not visible to end users yet. To see the new page, please replace import AccountPage from "./pages/Accounts/Account"; with import AccountPage from "./pages/Account/Index"; in ExplorerRoutes.tsx. We will make the new page visible to users when the development is done.

Reply:
Got it, when I looked at it the code was like this:

```
// import AccountPage from "./pages/Accounts/Account"; // import AccountPage from "./pages/Account/Index";
import AccountPage from "./pages/Account/Index";
```

Reply:
Sorry, I forgot to change it back in the last version. It should be correct now.